pilot/pkg/bootstrap/server.go

```diff
@@ -797,7 +801,21 @@ func (s *Server) initRegistryEventHandlers() {
 	s.ServiceController().AppendServiceHandler(serviceHandler)
 
 	if s.configController != nil {
-		configHandler := func(_, curr config.Config, event model.Event) {
+		configHandler := func(old config.Config, curr config.Config, event model.Event) {
+			if old.Generation == curr.Generation && curr.Generation != 0 {
+				// generation 0 indicates this is an ephemeral config generated by istio
+				if curr.GroupVersionKind == gvk.WorkloadEntry {
```

Comment:
Skipping WE doesn't seem like the right solution long term. Can we open an issue with a plan and link it here?

Reply:
I'm happy to open an issue, but I don't see other possibilities for solving this.  This logic skips sending pushes when status is updated *unless* the object is a WE, and the healthy condition has changed, in which case it continues with the push... Am I missing an opportunity for more refined logic?